std/go/grpc/servercore/mtls.go

```diff
@@ -33,10 +33,9 @@ func getMtlsConfig() (*tls.Config, error) {
 		return nil, err
 	}
 
+	caCertPem := os.Getenv("FOUNDATION_GRPCSERVER_CA_CERT")
 	pool := x509.NewCertPool()
-	for _, cert := range tb.CaChainPem {
-		pool.AppendCertsFromPEM([]byte(cert))
-	}
+	pool.AppendCertsFromPEM([]byte(caCertPem))
```

Comment:
I'd just inline it since we're not doing any checks whether the cert is there or not. The env var name makes it clear what we expect to find there.

```go
pool.AppendCertsFromPEM([]byte(os.Getenv("FOUNDATION_GRPCSERVER_CA_CERT")))
```

Reply:
done. I do not want to do any checks as its not required to provide a CA cert.